packages/terra-i18n/README.md

```diff
@@ -12,6 +12,7 @@ The terra-i18n component provides the internationalization to the React componen
 
 ## Getting Started
 
+At most time, you don't need to install this package directly. Please follow [this internationalization section](../../README.md#internationalizationi18n) to set up. `Base` will install this package for you.
```

Comment:
Then we could rephrase it like so

    Note: It is recommend to use `Base` instead of using this module, as base will initialize your internationalization for you

If that doesn't work, then I think we are documenting this at the wrong level. We would probably need to document it as a higher Getting Started with Terra section as opposed to putting it inside this module's documentation.